### PR TITLE
Update pull request template

### PR DESCRIPTION
**Pull request by daragu**
## *Important Read*

Update pull request template.

## What is the purpose of the pull request

`HudiSchemaEvolutionTest ` doesn't exist in this project.

## Brief change log

*(for example:)*
- *Fixed JSON parsing error when persisting state*
- *Added unit tests for schema evolution*

## Verify this pull request

*(Please pick either of the following options)*

This pull request is a trivial rework / code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

*(example:)*

- *Added integration tests for end-to-end.*
- *Added HudiSchemaEvolutionTest to verify the change.*
- *Manually verified the change by running a job locally.*

### CODE REVIEW

**Comment by daragu** [comment]
hi @the-other-tim-brown @jcamachor why workflow is awaiting approval from a maintainer?

https://github.com/apache/incubator-xtable/actions/runs/9816439820

**Comment by jcamachor** [comment]
@daragu , that seems related to the workflow runs approval process for the repo: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

**Review by jcamachor — APPROVED** [review]

**Pull request merged**
